src/main/java/core/basesyntax/store/strategy/TypeHandler.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.store.strategy;
+
+public interface TypeHandler {
+    void makeOperation(String fruitName, long quantity, int lineNumber);
```

Comment:
our handlers should have no idea of any lines
it doesn't know that we read information from file
```suggestion
    void makeOperation(String fruitName, long quantity);
```